libraries/AP_Motors/AP_MotorsTri.h

```diff
@@ -71,4 +71,7 @@ class AP_MotorsTri : public AP_MotorsMulticopter {
     float           _thrust_right;
     float           _thrust_rear;
     float           _thrust_left;
+
+    // reverse pitch
+    bool pitch_reversed = false;
```

Comment:
This should have an underscore prefix. Also no need to set it to false, it already is.

Reply:
fixed